miner/miner.go

```diff
@@ -390,8 +392,14 @@ func (m *Miner) mineOne(ctx context.Context, base *MiningBase) (*types.BlockMsg,
 		return nil, xerrors.Errorf("failed to compute winning post proof: %w", err)
 	}
 
+	ticketHash := blake2b.Sum256(ticket.VRFProof)
+	ticketNum := types.BigFromBytes(ticketHash[:]).Int
+	ticketDenu := big.NewInt(1)
```

Comment:
ideally the 'ticket to fraction' logic can be pulled into a separate function somewhere